src/services/package.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "@bcrs-shared-components/services",
+    "version": "1.0.0",
```

Comment:
Set this to 0.0.0. [ref](https://github.com/bcgov/bcrs-shared-components#how-to-create-a-new-package-you-depend-on-eg-new-mixins-folder)

Also please add the dependencies and devDependencies here (Axios, shared interfaces, etc).